### DISCUSSION TITLE
Fraud, Waste, and Abuse Policy

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#76] [Qualified]
### What are some documents containing these terms?

| Example | Service Name | URL | Title | Jurisdiction(s) |
|:----|:----|:----|:------|:-------------|
| 1 | Regence | [URL 🔗](https://www.regence.com/fraud) | Fraud & Abuse | 🇺🇸 |
| 2 | USA Health | [URL 🔗](https://www.usahealthsystem.com/fraud-and-abuse-policy) | Fraud and Abuse Policy | 🇺🇸 |
| 3 | Johns Hopkins Health | [URL 🔗](https://www.hopkinsmedicine.org/johns-hopkins-health-plans/providers-physicians/health-care-fraud-and-abuse) | Health Care Fraud and Abuse | 🇺🇸 |
| 4 | Columbia University | [URL 🔗](https://www.compliance.cuimc.columbia.edu/federal-state-regulations/fraud-abuse-policy) | Fraud & Abuse Policy | 🇺🇸 |
| 5 | Pittsburg State University | [URL 🔗](https://www.pittstate.edu/president/policies/fraud-waste-and-abuse-policy.html) | Fraud, Waste, and Abuse Policy | 🇺🇸 |
| 6 | University of Tennessee | [URL 🔗](https://policy.tennessee.edu/policy/fi0130-fraud-waste-and-abuse/) | Fraud, Waste and Abuse | 🇺🇸 |
| 7 | SJC COCC | [URL 🔗](https://stjohnsclerk.com/fraud-abuse-waste/fwa-policy/) | Fraud, Waste and Abuse Policy | 🇺🇸 |
| 8 | Erie Metropolitan Transit Authority | [URL 🔗](https://ride-the-e.com/wp-content/uploads/2023/03/Fraud-Waste-Abuse-Policy.pdf) | Fraud, Waste, and Abuse Policy | 🇺🇸 |

### What could be some alternative names for this type?

Fraud and Abuse Policy

### Whose rights and duties are defined in these terms?

staff, student, employees, contractors and vendors conducting business and others who use service resources or information with or without authorization

### What is the object of commitment in these terms?

define fraud, waste, and abuse and the procedures and responsibilities for preventing, reporting, investigating, and resolving known or suspected instances of such behavior

### Are there any relevant references for this type?

[USA Deficit Reduction Act of 2005](https://www.bswllp.com/the-5-million-question-are-you-complying-with-section-6032-of-the-dra)
[Fraud, Waste, or Abuse Policy — U.S. Courts](https://www.uscourts.gov/rules-policies/judiciary-policies/fraud-waste-or-abuse-policy)

**Comment by Ndpnt** [comment]
Does this terms type only available in US jurisdiction? Could you find examples in another jurisdiction?
